src/config-import.js

```diff
@@ -270,6 +281,27 @@ async function findTranslationsFile(entries) {
   return result
 }
 
+/**
+ * @param {ReadonlyArray<Entry>} entries
+ * @returns {Promise<MetadataFile | undefined>}
+ */
+async function findMetadataFile(entries) {
+  const metadataEntry = entries.find(
+    (entry) => entry.filename === 'metadata.json'
+  )
+  if (!metadataEntry) return
+  //assert(metadataEntry, 'Zip file does not contain metadata.json')
```

Comment:
since metadata, translations and presets files are loaded in parallel (see line 52), failing where it can't find a `metadata.json` file messes assertions in tests.